packages/web-components/src/component-mixins/cta/cta.ts

```diff
@@ -324,6 +332,46 @@ const CTAMixin = <T extends Constructor<HTMLElement>>(Base: T) => {
       }
     }
 
+    /**
+     * Check the URL for a fragment including the video id.
+     *
+     * If we find a URL fragment that includes the video id, we trigger the
+     * eventRunAction event, which for video will open the video and start
+     * playback in a lightbox. This is the same thing that happens when the user
+     * clicks on the CTA.
+     */
+    _checkUrlVideoTrigger() {
+      const { ctaType, disabled, href, videoDescription, videoName } = this;
+      // Without a video id, or if the button is disabled, there is nothing to
+      // do here.
+      if (ctaType !== CTA_TYPE.VIDEO || !href || disabled) {
+        return;
+      }
+      // Only trigger for the first CTA with the video id in the page.
+      if (this.ownerDocument.querySelector(`[href='${href}']`) !== this) {
+        return;
+      }
+      const { eventRunAction } = this.constructor as typeof CTAMixinImpl;
+      const hash = window.location.hash;
```

Comment:
Small thing but this will fail in SSR applications (window undefined). Can use [window-or-global](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/packages/web-components/src/components/masthead/masthead-composite.ts#L16).

Reply:
Thanks @kennylam . Fixed in recent commit https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/11253/commits/983f98f1c85158b67e1b5d657e2896f8dbf333da. 

Also noticed a JS error while testing. Not related to changes, but felt small enough to take care of here (https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/11253/commits/d815657486af3ac10c7e647326d3dfb1c8b2b528).